src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/ILanguageServiceHost.cs

```diff
@@ -33,5 +34,21 @@ IWorkspaceProjectContext ActiveProjectContext
         {
             get;
         }
+
+        /// <summary>
+        ///     Gets the object that represents the IVsENCRebuildableProjectCfg2.
+        /// </summary>
+        EncInterop.IVsENCRebuildableProjectCfg2 ENCProjectConfig2
```

Comment:
Sorry. I should have clarified. 

This must be object - this is a non-VS layer, make it _exactly_ like the error report, return object. I don't think we need one for all versions of the interface, and call it something like HostSpecificEditAndContinueService or something like that.